cli/error.go

```diff
@@ -8,10 +8,6 @@ import "strings"
 type Errors []error
 
 func (errList Errors) Error() string {
-	if len(errList) < 1 {
```

Comment:
This may be unnecessary in terms of the end result, but I suspect it was added to avoid the unnecessary time spent in the following lines by just returning immediately.